cosmo/serializer.py

```diff
@@ -62,6 +70,12 @@ def render_addresses(self):
 class RouterSerializer:
 
     def __init__(self, device, l2vpn_list, vrfs):
+        copy = device
+        for k in ["platform", "manufacturer", "slug"]:
```

Comment:
This can also be abstracted somehow. We probably need deep checks more than once later on.

Reply:
There might also be a lib for that.

Reply:
used a try / except instead. for now I don't really have a systemic solution for this, but for sure we can deal with that during the refactoring